content/docs/lifting-state-up.md

```diff
@@ -169,43 +169,43 @@ class TemperatureInput extends React.Component {
 
-Let's see how this works step by step.
+בואו נראה איך זה עובד צעד אחר צעד.
 
-First, we will replace `this.state.temperature` with `this.props.temperature` in the `TemperatureInput` component. For now, let's pretend `this.props.temperature` already exists, although we will need to pass it from the `Calculator` in the future:
+ראשית, אנו מחליפים את `this.state.temperature` עם `this.props.temperature` בקומפוננטת `TemperatureInput`. לעת עתה, בואו נמשיך להעמיד פנים ש`this.props.temperature` כבר קיים, למרות שנצטרך להעביר אותו מ-`Calculator` בעתיד:
 
 ```js{3}
   render() {
-    // Before: const temperature = this.state.temperature;
+    // קודם לכן: const temperature = this.state.temperature;
     const temperature = this.props.temperature;
     // ...
 ```
 
-We know that [props are read-only](/docs/components-and-props.html#props-are-read-only). When the `temperature` was in the local state, the `TemperatureInput` could just call `this.setState()` to change it. However, now that the `temperature` is coming from the parent as a prop, the `TemperatureInput` has no control over it.
+אנחנו יודעים ש-[props הם לקריאה בלבד](/docs/components-and-props.html#props-are-read-only). כאשר ה-`temperature` היה ב-state המקומי, ה-`TemperatureInput` יכל פשוט לקרוא ל-`this.setState()` כדי לשנות אותו. למרות זאת, עכשיו כאשר `temperature` מגיע מההורה בתור prop, ל-`TemperatureInput` אין שליטה עליו.
 
-In React, this is usually solved by making a component "controlled". Just like the DOM `<input>` accepts both a `value` and an `onChange` prop, so can the custom `TemperatureInput` accept both `temperature` and `onTemperatureChange` props from its parent `Calculator`.
+ב-React, זה בדרך כלל נעשה על ידי הפיכת קומפוננטה ל-"נשלטת". בדיוק כמו ב-DOM `<input>` מקבל גם `value` וגם prop של `onChange`, כך יכול גם ה-`TemperatureInput` המותאם אישית לקבל גם `temperature` וגם props של `onTemperatureChange` מההורה שלו `Calculator`.
 
-Now, when the `TemperatureInput` wants to update its temperature, it calls `this.props.onTemperatureChange`:
+עכשיו, שה-`TemperatureInput` רוצה לעדכן את הטמפרטורה שלו, הוא יקרא ל-`this.props.onTemperatureChange`:
 
 ```js{3}
   handleChange(e) {
-    // Before: this.setState({temperature: e.target.value});
+    // קודם לכן: this.setState({temperature: e.target.value});
     this.props.onTemperatureChange(e.target.value);
     // ...
 ```
 
->Note:
+>שימו לב:
 >
->There is no special meaning to either `temperature` or `onTemperatureChange` prop names in custom components. We could have called them anything else, like name them `value` and `onChange` which is a common convention.
+>אין משמעות מיוחדת לשמות ה-props `temperature` או `onTemperatureChange` בקומפוננטות מותאמות אישית. יכולנו לקרוא להם כל דבר אחר, כמו לקרוא להם `value` ו-`onChange` שהיא קונבנציה נפוצה.
```

Comment:
Convention in Hebrew is דרך ביצוע מקובלת, I think it is better.

Reply:
@itayyehezkel I think you suggested translation is a little bit overcomplicated. According to [Morifx](http://www.morfix.co.il/Convention) the 2 most viable options are קונבנציה or מוסכמה. I think the first is more widely used in this context, while the latter sounds a little more archaic. Let me know what you think. Thanks!

Reply:
I prefer מוסכמה because it is more understandable, but I know that most people using קונבציה.
Leave it open, and I will add it to the style guide discussion. #22